release.nix

```diff
@@ -126,6 +126,8 @@ let
     [ "cardano-node-profiled" "components" "exes" ]
     [ "checks" "tests" "cardano-node-chairman" ]
     [ "workbench" ] [ "profiles" ]
+    [ "haskellPackages" "cardano-node-chairman" "checks" ] [ "checks" "tests" "cardano-node-chairman" ]
+    [ "haskellPackages" "tx-generator" ]
```

Comment:
@MarcFontaine, re `musl` issues -- try to change this line to 
`[ "tx-generator" "components" "exes" ]`

Reply:
Ok, we found the right incantation.